docs/source/AdministratorGuide/Tutorials/dmsWithTs.rst

```diff
@@ -12,31 +12,427 @@ You should:
-By the end of the tutorial, you will be able to:
+The aim of the tutorial is to demonstrate how large scale data management operations (removals, replications, etc.) can
+be achieved using the Transformation System.  By the end of the tutorial, you will be able to:
 
 * Submit simple transformation for manipulating a given list of files
 * Have transformations automatically fed thanks to metadata
-* Write your own plugin for TransformationSystem
+* Write your own plugin for the TransformationSystem
 
+The transformations can be monitored and controlled with the ``Transformation Monitor`` in the ``WebApp`` when you use
+the ``dirac_prod`` group.
 
-More links
+
+More Links
 ==========
 
 * :ref:`adminTS`
 
 
-Installing the RequestManagementSystem
-======================================
+Creating a Replication Transformation
+=====================================
+
+.. highlight:: console
+
+This section is to be performed as ``diracuser`` with a proxy in ``dirac_prod`` group.
+
+First we need to create some files and upload them to ``StorageElementOne``::
+
+  [diracuser@dirac-tuto ~]$ for ID in {1..10}; do echo "MyContent $ID" > File_${ID} ; dirac-dms-add-file /tutoVO/data/Trans_01/File_${ID} File_${ID} StorageElementOne ; done
+
+Then we create the list of LFNs we just uploaded::
+
+  [diracuser@dirac-tuto ~]$ dirac-dms-user-lfns -b /tutoVO/data/Trans_01
+  Will search for files in /tutoVO/data/Trans_01
+  /tutoVO/data/Trans_01: 10 files, 0 sub-directories
+  10 matched files have been put in tutoVO-data-Trans_01.lfns
+
+The easiest way to create a transformation to replicate files is by using the ``dirac-transformation-replication``
+command::
+
+  [diracuser@dirac-tuto ~]$ dirac-transformation-replication 0 StorageElementTwo --SourceSEs StorageElementOne -x
+  Created transformation NNN
+  Successfully created replication transformation
+
+This created transformation with the unique transformation ID *NNN* (e.g., 1).
+
+By default this transformation uses *Metadata* information to obtain the input files using the ``InputDataAgent``. Instead
+we can also just add files manually using the list we created previously, replace NNN by the ID of the transformation
+that was just created::
+
+  [diracuser@dirac-tuto ~]$ dirac-transformation-add-files NNN tutoVO-data-Trans_01.lfns
+  Successfully added 10 files
+
+
+Now we have to wait until the ``TransformationAgent`` runs again and creates a *Task* for each of the files. Once the
+tasks are created, the ``RequestTaskAgent`` creates a request out of each task, which is then processed in the
+``RequestExecutingAgent`` of the RMS.
+
+
+Creating a Replica Removal Transformation
+=========================================
+
+
+In this step we want to remove the replicas of our files from ``StorageElementOne``, for this purpose we have to write a
+script that creates a removal transformation:
+
+.. code-block:: python
```

Comment:
Put this part in a Developer tutorials

Reply:
Was it really this part we wanted to move as well?
How to submit a transformation via script doesn't sound to developer like, so I would rather leave this here.

Reply:
I think we wanted that to go in the developer doc, and the admin doc to have only the version with the script. But if I remember correctly, the script is not ready for that yet

Reply:
Also that, yes, the dirac-transformation-replication does not support the RemoveReplica operation just by itself, only in combination with ReplicateAndRegister (to move files from A to B).

I think this example is closer to the Ops part in the DevOps grey area